src/backbone.service.js

```diff
@@ -1,11 +1,39 @@
-import classify from 'backbone-metal-classify';
 import Radio from 'backbone.radio';
 import _ from 'underscore';
 import PromisePolyfill from 'es6-promise';
 
 const resolved = PromisePolyfill.Promise.resolve();
 
-Radio.Channel = classify(Radio.Channel);
+//copy of Backbone.js extend function
+Radio.Channel.extend = function(protoProps, staticProps) {
```

Comment:
Maybe instead of copying whole extend function it can be done like this
`Radio.Channel.extend = Backbone.Model.extend`

But I don't know. I don't like neither solution in PR nor I wrote above...

Reply:
For the record, i forked it as radio.service and modernized its codebase: https://github.com/blikblum/radio.service